### ISSUE TITLE
Problems with train_mask

### ISSUE THREAD

**Issue by AdaxKK** [silenceFS/LINAS#1]
Thank you for your work! When train_mask is in the project, val_len has no corresponding value, what should I set it to? Also, when calling train_mask, do we use the data from the training set or the validation set?
